server/sonar-server/src/main/java/org/sonar/server/computation/step/PurgeDatastoresStep.java

```diff
@@ -42,14 +43,16 @@
   private final DbIdsRepository dbIdsRepository;
   private final TreeRootHolder treeRootHolder;
   private final SettingsRepository settingsRepository;
+  private final DisabledComponentsHolder disabledComponentsHolder;
 
   public PurgeDatastoresStep(DbClient dbClient, ProjectCleaner projectCleaner, DbIdsRepository dbIdsRepository, TreeRootHolder treeRootHolder,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)